CONTRIBUTING.md

```diff
@@ -8,152 +8,104 @@
+# Open Development
+All contributors use GitHub’s pull request (PR) workflow to submit changes. Before merging, all pull requests require two approvals.
 
-## Ways to contribute
+# Semantic Versioning
+VDK follows semantic versioning standards as part of https://semver.org/.
 
-We welcome many different types of contributions and not all of them need a Pull request. Contributions may include:
+Given a version number MAJOR.MINOR.PATCH, increment the:
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backwards compatible manner
+- PATCH version when you make backwards compatible bug fixes
 
-* New features and proposals
-* Documentation
-* Bug fixes
-* Issue Triage
-* Answering questions and giving feedback
-* Helping to onboard new contributors
-* Other related activities
+# Bugs
 
-# Getting Started
+## Where to Find Known Issues
+Review the [list of existing issues](https://github.com/vmware/versatile-data-kit/issues) to see if the bug has already been reported.
 
-## Contribution Flow
+## Reporting New Issues
+To report a bug, [create a GitHub issue](https://github.com/vmware/versatile-data-kit/issues). Include steps to replicate the bug in the issue description.
 
-This is a rough outline of what a contributor's workflow looks like:
+## Security Bugs
+If you believe you have found a security bug, [create a GitHub issue](https://github.com/vmware/versatile-data-kit/issues) and reach out to us in the [Contacts Us](#contact-us) section.
 
-* Use forks to only contribute changes on examples and documentation.
-   * Currently we accept contribution from forks only for examples and documentation changes until PR 854 is fixed. Until then please request write priveleges and create a branch in the main repo as described below.
-* Create a topic branch from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
-* Make commits of logical units
-* Make sure your commit messages are with the proper format, quality and descriptiveness (see below)
-* Push your changes to the topic branch in your fork
-* Create a pull request containing that commit
+# Propose a Change
+Before suggesting a feature or change, consider the impact of this change. Does it only serve your needs, or does it serve the broader community's needs?
 
-We follow the GitHub workflow and you can find more details on the [GitHub flow documentation](https://docs.github.com/en/get-started/quickstart/github-flow).
+For substantial features or changes, submitting your design is valuable. Creating a PR in Github with your proposal as markdown in the [specs](https://github.com/vmware/versatile-data-kit/blob/main/specs) directory is recommended. Reviews, feedback, and approvals are documented in the PR.
```

Comment:
nit: Would really like to highlight on VDK Enhancement Proposal or VEP. For example https://github.com/python/peps, or https://jcp.org/en/jsr/overview

Reply:
@ivakoleva Is there a VEP template they can reference or a sample VEP we can link to?

Reply:
@gary-tai Take a quick look at the specifications directory https://github.com/vmware/versatile-data-kit/tree/main/specs (- unfortunately there isn't any match when searching by "spec" in Github Wiki yet). There is, however, a very detailed template with content structure expectations, and a few full-length specifications now. Those are meant to be alive and reflecting the current feature or vice-versa 😊